german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
Ja stimmt in der 1. noch nicht...
Für die Erste wäre jetzt in Griffweite usw die Optimallösung. Das haben die Jungs von System Matters für Dungeon World so ähnlich gemacht. Wobei ich halt da ein Problem bei Griffweite und Reichweite habe. Das bedeutet im Grunde dasselbe.

Der wirkliche Kopfschmerzen Verursacher ist ja im Grunde nur das passende Äquivalent für close.

Also ich wäre für die sprachliche Variante. Wir haben ja festgestellt, dass es im Original ja genau darum gehen soll. Dann werden es halt mehr als ein Wort aber dafür halt in den dazugehörigen Sätzen sinniger. Aber es sollten Entfernungsbegriffe sein.
Dicht dran klingt im Satz auch blöd. Nah und dann in der Nähe ist ähnlich wie mit Weit und Weit entfernt. Nah, Weit, Fern wären einfach und mit den Sätzen kompatibel, aber dann fehlt eine Kategorie. Setzt man dann etwas längeres ein, wirkt es doof.

Dicht dran, in der Nähe, Weit weg, in der Ferne ist so gesehen vorerst die bessere Lösung.